app/config.py

```diff
@@ -1,20 +1,13 @@
-
 LCS_BASE_URL = ""
```

Comment:
no changes were made to this file except for formatting, so let's remove it from this PR. In the future we'll only have a `config.example.py` file.